src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -109,7 +109,7 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
-        Birthday updatedBirthday = editPersonDescriptor.getBirthday().orElse(personToEdit.getBirthday());
+        Birthday updatedBirthday = editPersonDescriptor.getBirthday().orElse(personToEdit.getBirthday().orElse(null));
```

Comment:
Just a bit unsure if using `null` is a good idea, would using `Optional` or empty string be better?

Reply:
I will have to store the birthday attribute as a null anyways, so I don't think it adds any value to use optional.